LICENSE

```diff
@@ -290,8 +290,8 @@ to attach them to the start of each source file to most effectively
 convey the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    {description}
-    Copyright (C) {year}  {fullname}
+    <one line to give the program's name and a brief idea of what it does.>
```

Comment:
You should probably fill this in.

Reply:
It is not meant to be filled in, it's simply a part of the license that informs distributors of changed code how they may wish to describe the changed program and add a copyright statement, as described in https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLOmitPreamble

Reply:
Oh cool, TIL.

Reply:
That is why, perhaps somewhat amusingly, every GPL2.0 license mentions "Ty Coon, President of Vice"